[2025-02-19T20:12:04.534+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-19T20:12:04.544+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag_example_v0.first_task scheduled__2025-01-19T00:00:00+00:00 [queued]>
[2025-02-19T20:12:04.549+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag_example_v0.first_task scheduled__2025-01-19T00:00:00+00:00 [queued]>
[2025-02-19T20:12:04.549+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-19T20:12:04.555+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): first_task> on 2025-01-19 00:00:00+00:00
[2025-02-19T20:12:04.559+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1257) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-19T20:12:04.560+0000] {standard_task_runner.py:72} INFO - Started process 1258 to run task
[2025-02-19T20:12:04.562+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'first_dag_example_v0', 'first_task', 'scheduled__2025-01-19T00:00:00+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/wtv.py', '--cfg-path', '/tmp/tmptqqwx59g']
[2025-02-19T20:12:04.566+0000] {standard_task_runner.py:105} INFO - Job 27: Subtask first_task
[2025-02-19T20:12:04.599+0000] {task_command.py:467} INFO - Running <TaskInstance: first_dag_example_v0.first_task scheduled__2025-01-19T00:00:00+00:00 [running]> on host 8fb7a5664bcb
[2025-02-19T20:12:04.647+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='goncalo' AIRFLOW_CTX_DAG_ID='first_dag_example_v0' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-19T00:00:00+00:00'
[2025-02-19T20:12:04.648+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-19T20:12:04.656+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-19T20:12:04.657+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Hello world!"']
[2025-02-19T20:12:04.663+0000] {subprocess.py:99} INFO - Output:
[2025-02-19T20:12:04.664+0000] {subprocess.py:106} INFO - Hello world!
[2025-02-19T20:12:04.664+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-19T20:12:04.688+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-19T20:12:04.688+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=first_dag_example_v0, task_id=first_task, run_id=scheduled__2025-01-19T00:00:00+00:00, execution_date=20250119T000000, start_date=20250219T201204, end_date=20250219T201204
[2025-02-19T20:12:04.713+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-19T20:12:04.727+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-19T20:12:04.729+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
